packages/avatars/demo/statusindicator.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { StatusIndicator } from '@zendeskgarden/react-avatars';
+
+import { StatusIndicatorStory } from './stories/StatusIndicatorStory';
+
+<Meta title="Packages/Avatars/StatusIndicator" component={StatusIndicator} />
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="StatusIndicator"
+    argTypes={{
+      caption: { name: 'caption', control: 'text', table: { category: 'Story' } },
+      show: {
+        name: 'show',
+        control: 'radio',
+        options: ['default caption', 'set caption', 'no caption'],
+        table: { category: 'Story' }
+      }
+    }}
+    args={{
+      show: 'default caption'
+    }}
+    parameters={{
+      design: {
+        allowFullscreen: true,
+        type: 'figma',
+        url: 'https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=102%3A110'
+      }
+    }}
```

Comment:
This addition should be delayed until `StatusIndicator` is on the main Figma branch.

Reply:
Yes this is a placeholder - please see the PR description under `TODO`. I can remove it for now.